### ISSUE TITLE
Widget giving error with Mendix 7.18

### ISSUE THREAD

**Issue by heuvelb** [mendixlabs/listview-swipe#7]
On Mendix 7.18, the implementation of the widget gives an error in the frontend:
"No constructor found for widget ListViewSwipe.widget.ListViewSwipe"

On the Mendix forum, similar issues are knows with other widgets. The provided solutions for those problems don't seem to solve this issue:
- clean deployent directory;
- empty cache;
- reinstall widget

Might there be an compatibility issue with the new 7.18 version?

**Comment by McDoyen** [comment]
HI@heuvelb, 
We were unable to reproduce the issue.

Please check that you are you are using the latest version of List view swipe.
If possible please provide a sample project in which you are getting this error.

Cheers,
Paul

**Comment by McDoyen** [comment]
Hi @heuvelb,

Here is a [test project](https://github.com/FlockOfBirds/listview-swipe/releases/download/v1.1.0-beta/TestListviewSwipeOut.mpk) running on mx 7.18 that I tested with https://github.com/FlockOfBirds/listview-swipe/releases/download/v1.1.0-beta/TestListviewSwipeOut.mpk

The [widget](https://github.com/FlockOfBirds/listview-swipe/releases/download/v1.1.0-beta/ListviewSwipe.mpk) being used is a beta version. Please test with it too to see if it fixes your issue.

Kind regards,
Paul

**Comment by heuvelb** [comment]
Hi Paul,

 

Thank you for the update. I’ve cleaned the deployment folder, deleted the old widget, refreshed project folder and installed the beta version… to no avail.

I am still getting the no constructor found error.

 

Can I assist you in helping to debug this?

 

Regards, Bart van den Heuvel

**Comment by McDoyen** [comment]
Dear @heuvelb,
Kindly try reproducing the issue from with in the sample project I sent you. That's the best way we can help for now.

Regards
Paul

**Comment by heuvelb** [comment]
Hi Paul,

 

With your project the widget is functioning like a charm. I can’t seem to replicate my error.

The only difference I spotted was that in your situation, the listview was filled through a microflow, so I changed my retrieve method too. This also disabled the search function on the listview. I thought that might have caused the issue. 

One thing I did notice in my app, was that the listview objects were rendered slightly blurred, even though the widget gave the error. Removing the widget from the page made the text appear crisp again.

 

I understand this is a very undefined situation for you, so if you would like to, I am more than happy to assist you.

 

Kind regards, Bart van den Heuvel
